src/main/java/com/google/sps/utils/Game.java

```diff
@@ -0,0 +1,67 @@
+package com.google.sps.utils;
+import com.google.appengine.api.datastore.Entity;
+import java.lang.IllegalArgumentException;
+import java.util.List;
+import java.util.ArrayList;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+ 
+public final class Game {
```

Comment:
I would maybe make a test for all the setters and getters

Reply:
Alright, I made one for this!